capstone-server/src/main/java/com/hcl/capstoneserver/config/GlobalErrorHandler.java

```diff
@@ -62,14 +66,13 @@
     }
 
     @ExceptionHandler(BadCredentialsException.class)
-    public final ResponseEntity<DefaultErrorResponse> handleBadCredentials(BadCredentialsException ex) {
+    public final ResponseEntity<DefaultErrorResponse> handleBadCredentials() {
```

Comment:
```suggestion
public final ResponseEntity<DefaultErrorResponse> handleBadCredentials(BadCredentialsException ex) {
```

keep the original structure

Reply:
But ex is never used.